### PR TITLE
refactor(core): Update rule typescript-eslint/no-unused-vars to not error when variable starts with _

### PR DESCRIPTION
**Pull request by michael-radency**

### CODE REVIEW

File: packages/nodes-base/credentials/Aws.credentials.ts [hunk 1]
```diff
@@ -370,7 +370,7 @@ export class Aws implements ICredentialType {
 	};
 }
 
-function queryToString(params: IDataObject) {
+function _queryToString(params: IDataObject) {
```
Comment (netroy) [inline packages/nodes-base/credentials/Aws.credentials.ts:373]: why not delete this? looks unused.

File: packages/nodes-base/nodes/Airtable/Airtable.node.ts [hunk 2]
```diff
@@ -2,21 +2,23 @@ import { IExecuteFunctions } from 'n8n-core';
...
 	INodeType,
 	INodeTypeDescription,
 	NodeOperationError,
 } from 'n8n-workflow';
 
 import { apiRequest, apiRequestAllItems, downloadRecordAttachments } from './GenericFunctions';
 
+// never used
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 interface AirtableBase {
```
Comment (netroy) [inline packages/nodes-base/nodes/Airtable/Airtable.node.ts:15]: should we delete these?

File: packages/nodes-base/nodes/Airtable/GenericFunctions.ts [hunk 3]
```diff
@@ -36,7 +36,7 @@ export async function apiRequest(
 	option: IDataObject = {},
 	// tslint:disable-next-line:no-any
 ): Promise<any> {
-	const credentials = await this.getCredentials('airtableApi');
+	const _credentials = await this.getCredentials('airtableApi');
```
Comment (netroy) [inline packages/nodes-base/nodes/Airtable/GenericFunctions.ts:39]: we should delete this line.

**Review by netroy — APPROVED** [review]
Thanks for cleaning up

**Comment by janober** [comment]
Got released with `n8n@0.202.0`

**Pull request merged**